scripts/detectNotUsingGitPush1by1.fsx

```diff
@@ -19,6 +19,37 @@ if String.IsNullOrEmpty githubEventPath then
 
     Environment.Exit 2
 
+let accessToken = Environment.GetEnvironmentVariable "ACCESS_TOKEN"
+(*
+To save your GitHub access token as an environment variable in GitHub Actions, you can use the secrets feature in GitHub.
+
+Here are the steps to save your access token as an environment variable:
+
+1. Generate a personal access token on GitHub (if you haven't already done so).
+2. Go to the settings for your repository and click on "Secrets and variables" in the left-hand sidebar.
+3. Click on the "Actions" under "Secrets and variables".
+4. Click on the "New secret" button.
```

Comment:
New secret -> New repository secret

Reply:
With secrets.GITHUB_TOKEN we don't need those lines anymore